index.js

```diff
@@ -36,8 +36,46 @@ module.exports = {
   },
 
   async updateDeps(options) {
+    let manifestPath = join(options.target, 'package.json');
+    let manifestBuffer = await readFile(manifestPath);
+    let manifest = JSON.parse(manifestBuffer.toString());
+
+    let existingDeps = [
+      ...Object.keys(manifest.dependencies || {}),
+      ...Object.keys(manifest.devDependencies || {}),
+    ];
+
+    let ensureLatestDeps = [
+      'eslint',
+      'eslint-plugin-ember',
+      'eslint-plugin-n',
+      '@babel/eslint-parser',
+    ];
+
     // this.addPackagesToProject doesn't respect the packageManager that the blueprint specified 🙈 so we're skipping a level here
     let installTask = this.taskFor('npm-install');
+    let uninstallTask = this.taskFor('npm-uninstall');
+
+    await uninstallTask.run({
+      'save-dev': true,
+      verbose: false,
+      packages: [
+        // Not needed anymore
+        'ember-fetch',
+        'broccoli-asset-rev',
+        'ember-cli-app-version',
+        'ember-cli-clean-css',
+        'ember-cli-dependency-checker',
+        'ember-cli-sri',
+        'ember-cli-terser',
+
+        ...ensureLatestDeps,
+        // Linting
+        '@babel/plugin-proposal-decorators',
```

Comment:
Unlike upstream, we don't want this plugin because we're using `decorator-transforms` in our babel config.